mod/search.php

```diff
@@ -214,7 +214,7 @@ function search_content(App $a) {
 			FROM `item` %s
 			WHERE %s AND (`item`.`uid` = 0 OR (`item`.`uid` = %s AND NOT `item`.`global`))
 				$sql_extra
-			GROUP BY `item`.`uri` ORDER BY `item`.`id` DESC LIMIT %d , %d",
+			GROUP BY `item`.`uri`, `item`.`id` ORDER BY `item`.`id` DESC LIMIT %d , %d",
```

Comment:
My suggestion was very indeed generic and at the end of an exhausting review process, so it definitely can't apply as is to this query. However there's no way around it, this group by isn't working either.

I have no good easy simple solution to offer you, the query clearly wasn't designed with respect to SQL92, and we are left with the ungrateful job of fixing stuff.